.github/workflows/go.yml

```diff
@@ -90,8 +77,8 @@ jobs:
     - name: Build Antrea binaries
       run: GOARCH=${{ matrix.arch }} make bin
 
-  windows-bin:
-    name: Build Antrea Windows binaries
+  windows-bin-and-antctl:
+    name: Build Antrea Windows and Antctl binaries
```

Comment:
I refined this part and move antctl to step `bin` to make it more clear, and remove the `matrix` in `bin` considering it will create three different jobs for different matrix value which is unnecessary due to all `make bin` is actually run in the same ubuntu platform.